realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -305,6 +306,41 @@ public void removeChangeListener(OrderedRealmCollectionChangeListener<RealmResul
     }
 
     /**
+     * Returns an Rx Observable that monitors changes to this RealmResults. It will emit the current RealmResults when
+     * subscribed to. For each update to the RealmResult a pair consisting of the RealmResults and the
```

Comment:
picky, but you could leave the "to" off the end of this sentence, if you wanted.